ci: add changelog to image label when pushing new versions

jgrer opened the pull request:
- Changed release.yaml to gather the auto generated release notes into release_notes.md
- Added code to convert release_notes.md into a html file
- Passed this html file to the dockerfile for the extension using build args
- Set this html file in string format as the com.docker.extension.changelog parameter

ashnamehrotra commented on Makefile:
```diff
@@ -1,6 +1,7 @@
-IMAGE?=projectcopacetic/copacetic-docker-desktop-extension
+IMAGE?=jgrerr/copacetic-docker-desktop-extension
```
```suggestion
IMAGE?=projectcopacetic/copacetic-docker-desktop-extension
```

ashnamehrotra commented:
@jgrer can we also change the version in the install link in the readme on release? `docker extension install projectcopacetic/copacetic-docker-desktop-extension:0.1.0`

jgrer commented:
> @jgrer can we also change the version in the install link in the readme on release? `docker extension install projectcopacetic/copacetic-docker-desktop-extension:0.1.0`

removed the example of the install command (not worth to update) and updated the link to point to the latest version of the extension

ashnamehrotra reviewed (approved):
LGTM
